pages/requests/new/[ware].js

```diff
@@ -166,6 +170,14 @@ const NewRequest = () => {
             requestForm={requestForm}
             updateRequestForm={updateRequestForm}
           />
+          <Button
+            addClass='btn btn-primary my-4 ms-auto d-block'
+            backgroundColor={buttonBg}
+            disabled={buttonDisabled}
```

Comment:
does this still work if we pass the prop to StandardRequestOptions instead of duplicating the same button code in NewRequest?

Reply:
> The button is disabled, but because the page shows that it is loading, you can't really ever see it being disabled in the UI. not sure if we should still leave it there as a backup, or if its better to ONLY have the loading component while the submit is processing @alishaevn thoughts?

better yet, we possibly could remove the disable feature altogether for the reason you listed above. the goal of disabling was so the user couldn't click it multiple times. the loading spinner achieves that.

Reply:
> > Also, i thought it made sense to have buttonDisabled be a separate state than formSubmitting, because in a separate PR potentially the button should be disabled before the form is filled out/valid
> 
> good idea to keep them separate for the separate use case. however, a user can't submit a request without the required inputs being filled out anyway. and if we remove the "disabled" prop in favor of just showing the loading spinner...it renders the need for a disabled button obsolete.

at first i also thought it might make the disabled button obsolete, but then i realized that the `formSubmitting` needed to be moved down into the block that checked the validations, or else if the user clicked the button without filling anything out the form just got stuck in the loading state and never created a request like so:

https://share.getcloudapp.com/X6uKkQpx

so after moving the "formSubmitting" down, i was noticing it takes more time to get to that point, and there is still a split second in between where the user could hit that button twice and create a duplicate request (which is what happened to me below with these sets of duplicates) so i think we do still need to disable the button for that tiny  interim period

<img width="1363" alt="image" src="https://user-images.githubusercontent.com/73361970/221024223-3850e0aa-0c01-4dd4-82e3-0e657ec7c550.png">

<img width="1322" alt="image" src="https://user-images.githubusercontent.com/73361970/221023564-87448ae7-93cc-4457-a21b-820371794d62.png">